### PR TITLE
Update default_items.php

### PR DESCRIPTION
**Pull request by remotehelp**
"label label-warning" text "Unpublished" view only for administrator's and not need display for all user in filter label tag.

### CODE REVIEW

**Comment by therunnergit** [comment]
@remotehelp the code changes are really simple, but I cannot understand the description of the PR. Can you rephrase to clearly state what you want to fix?

**Comment by remotehelp** [comment]
@therunnergit text "Unpublished" beside field filter always display for all user include Guests - why?

**Comment by therunnergit** [comment]
According to your PR, it's a bug that needs to be fixed. 
@Test works fine.

**Comment by Kubik-Rubik** [comment]
Thank you @remotehelp! Merged.
